contracts/HeapOrdering.sol

```diff
@@ -12,12 +12,12 @@ library HeapOrdering {
     struct HeapArray {
         Account[] accounts; // All the accounts.
         uint256 size; // The size of the heap portion of the structure, should be less than accounts length, the rest is an unordered array.
-        mapping(address => uint256) ranks; // A mapping from an address to a rank in accounts. Beware: ranks are shifted by one compared to indexes, so the first rank is 1 and not 0.
+        mapping(address => uint256) indexes; // A mapping from an address to an index in accounts.
```

Comment:
Perhaps is it missing what you described in the PR: how do we access next, prev and parents based on the index of an account